src/components/LikeThought.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react'
```

Comment:
Imported useEffect but seems like it's not used in the component. so better to remove to keep the code clean :)